MAVProxy/mavproxy.py

```diff
@@ -1417,7 +1422,8 @@ def quit_handler(signum = None, frame = None):
         mpstate.rl.set_prompt("")
 
     # call this early so that logdir is setup based on --aircraft
-    (mpstate.status.logdir, logpath_telem, logpath_telem_raw) = log_paths()
+    if not opts.no_telemetry_logs:
```

Comment:
This leaves ``mpstate.status.logdir = None``, which can break other modules. In particular, combined with the ``--continue`` option.

This also stops other modules from automatically saving information (such as params, waypoints, fence).

Recommend removing this statement.

Reply:
we do check for logdir==None in a few places, eg. fence and param, likely just need a few more checks